src/components/m-dropdown/_template.js

```diff
@@ -3,6 +3,12 @@ import raw from 'nanohtml/raw';
 import classnames from 'classnames';
 
 const arrowIcon = '<axa-icon size="sm" icon="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
+const getSelectedItem = (items) => {
+  if (!items || !Array.isArray(items) || !items.length) {
+    return null;
+  }
+  return items.filter(item => item.isSelected)[0];
```

Comment:
if `return items.filter(item => item.isSelected)[0]` is undefined - I would recommend to return null
like that `return items.filter(item => item.isSelected)[0] || null` or a other implementation.
because in the render template you should do a  save type check like `getSelectedItem(items) === null`

Reply:
I removed this helper, but I return null in the duplicate helper and do type check.